bundlex.exs

```diff
@@ -11,14 +11,13 @@ defmodule Membrane.G711.FFmpeg.BundlexProject do
          "https://github.com/BtbN/FFmpeg-Builds/releases/download/latest/ffmpeg-n6.0-latest-linuxarm64-gpl-shared-6.0.tar.xz"}
 
       %{os: "linux"} ->
-        {:precompiled,
-         "https://github.com/BtbN/FFmpeg-Builds/releases/download/latest/ffmpeg-n6.0-latest-linux64-gpl-shared-6.0.tar.xz"}
+        "https://github.com/BtbN/FFmpeg-Builds/releases/download/latest/ffmpeg-n6.0-latest-linux64-gpl-shared-6.0.tar.xz"
```

Comment:
Shouldn't we modify the one for `aarch64/linux` as well?

Reply:
I copied that from [`membrane_h264_ffmpeg_plugin`](https://github.com/membraneframework/membrane_h264_ffmpeg_plugin/blob/85c920120d77426fe9187cc0f5edcc1679ef777a/bundlex.exs#L4-L25), but I can ask them why they didn't apply this modification also for arch.

Reply:
They answer is that this is probably a bug 😅